modules/spo-recording-profiles.adoc

```diff
@@ -111,26 +113,28 @@ $ oc -n openshift-security-profiles delete pod my-pod
 
 . Confirm the Security Profiles Operator reconciles the two {type} profiles:
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-$ oc -n openshift-security-profiles get sp
+$ oc get {object} -n openshift-security-profiles
```

Comment:
should be 
$ oc get {object} -n my-namespace

Reply:
nice catch. I have no more comments and overall this PR is a nice improvement!